docs/_data/sidebars/mydoc_sidebar.yml

```diff
@@ -138,9 +138,9 @@ entries:
     #   url: /observation.html
     #   output: web, pdf
 
-    # - title: Observation Period
-    #   url: /observation_period.html
-    #   output: web, pdf
+    - title: Observation Period
+      url: /observation_period_end_date.html
```

Comment:
rather than pointing directly to the obs. period end date convention page, this should point to the table-level page for the observation_period table.  

the observation_period page still needs to be added; you can add it under the observation_period folder following [this](https://github.com/OHDSI/Themis/blob/master/docs/pages/location/location.md?plain=1) example.

thanks!

Reply:
@katy-sadowski , I will work on it and submit the changes